### ISSUE TITLE
NTR: 3-hydroxyadipic acid high in urine (Lab: chemistry: organic acids)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8612]
New term request 3-hydroxyadipic acid high in urine (segal_030513181806):

**Comment by pnrobinson** [comment]
child of Increased urinary non-proteinogenic amino acid level

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000609
- label: Elevated urinary 3-hydroxyadipic acid level
- parents: [Dicarboxylic aciduria (HP:0003215)](https://hpo.jax.org/app/browse/term/HP:0003215)
- pmid: 
- def: The amount of 3-hydroxyadipic acid in the urine, normalized for urine concentration, is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8612
- synonyms: Elevated urinary 3-hydroxyadipate level(NONE)
